docs/grid-functionalities/excel-copy-buffer.md

```diff
@@ -94,4 +94,26 @@ this.gridOptions = {
     sanitizeDataExport: true
   },
 };
-```
\ No newline at end of file
+```
+
+### Disable pasting on specific columns
+If you want to disable pasting values for specific columns you can deactivate it using the denyPaste property on the Column config.
+
+```typescript
+this.columnDefinitions = [
+  { 
+    id: 'title', name: 'Title', field: 'id', 
```

Comment:
can we maybe use the same `Col A` as in your table below? Just to make it clearer